voting-dapp-template/move/Move.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "VotingTemplate"
```

Comment:
can we rename the folder name to `fungible-asset-voting` and package name to `FungibleAssetVoting`?  We can rename it back to template when we add to CAD (create-aptos-dapp) later.